.gitignore

```diff
@@ -1,2 +1,2 @@
 /vendor/
-/node_modules/
\ No newline at end of file
+/node_modules/
```

Comment:
this changes should be separate PR ok for now

Reply:
Okay